header-enricher-app-dependencies/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.springframework.cloud.stream.app</groupId>
+	<artifactId>header-enricher-app-dependencies</artifactId>
+	<version>1.0.0.BUILD-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>header-enricher-app-dependencies</name>
+	<description>Spring Cloud Stream header-enricher App Dependencies</description>
+
+	<parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.2.1.RELEASE</version>
+		<relativePath />
+	</parent>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud.stream.app</groupId>
+				<artifactId>spring-cloud-starter-stream-processor-header-enricher</artifactId>
+				<version>1.0.0.BUILD-SNAPSHOT</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+	<profiles>
```

Comment:
M-m-m. Couldn't we go an opposite way?
Remove it right now, merge and check how is it going on CI.
We always can restore this section with a subsequent commit if that.
At the same time removing it we will have shorter pom and easier review 😄 

Just IMHO. 
If you insist to leave I can live with that...